pronotepy/ent.py

```diff
@@ -6,6 +6,65 @@
 log = logging.getLogger(__name__)
 log.setLevel(logging.DEBUG)
 
+def ac_grenoble(username, password):
+    """
+    ENT ac Grenoble
+
+    Parameters
+    ----------
+    username : str
+        username
+    password : str
+        password
+    
+    Returns
+    -------
+    cookies : cookies
+        returns the ent session cookies
+    """
+    # ENT / PRONOTE required URLs
+    identity_provider = "https://cas.ent.auvergnerhonealpes.fr/login?selection=EDU&service=https://0380029A.index-education.net/pronote/&submit=Confirm"
+    login_service_provider = "https://educonnect.education.gouv.fr/idp/profile/SAML2/POST/SSO"
+    ent_login = ""
```

Comment:
useless line ? you already have it on [L47](https://github.com/bain3/pronotepy/pull/71/files#diff-6203c7f8202b3248dae5e1f35189b24f2ab11db401c8428a27f052ffa6ba0e58R47) and do not use it before it get set to the response url